analysis/experiment_results.py

```diff
@@ -31,8 +31,10 @@ class ExperimentResults:
     template, only the properties needed for the given report will be computed.
     """
 
+    # pylint: disable=too-many-arguments, too-many-instance-attributes
```

Comment:
nit: So i think this style of pylint directives will be applied to the entire file.
I think you might want to apply it just to the function so if you delete this directive and put it on the line where the complaint appears, that should apply it locally.